File: aspnet-core/src/EducationAdmin.Core/Education/LessonAttendance.cs
using Abp.Domain.Entities.Auditing;
using EducationAdmin.Sales;
using EducationAdmin.Students;
using System;
using System.Collections.Generic;
using System.Text;

namespace EducationAdmin.Education
{
    public class LessonAttendance : FullAuditedEntity<long>
    {
        public long OrderId { get; set; }

        public Order Order { get; set; }

        public long LessonId { get; set; }

        public Lesson Lesson { get; set; }

        public bool Attended { get; set; }
    }
}

File: vue/src/store/entities/lessonAttendance.ts
import Entity from './entity'
import Student from './student'
import Lesson from './lesson'
import Contract from './contract'
import Order from './order';

export default class LessonAttendance extends Entity<number>{

    lessonId: number
    lesson?: Lesson
    orderId: number
    order: Order
    attended: boolean
    lastModificationTime?: string
    lastModifierUserId?: number
    creationTime?: string
    creatorUserId?: number
    note?: string
    contract?: Contract

}